packages/dds/tree/src/test/testSrcPath.cts

```diff
@@ -0,0 +1,22 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "node:assert";
+import path from "node:path";
+
+// FUTURE: Without CommonJS requirement __dirname can be acquired from import.meta.url
+// In Node 20+ use:
```

Comment:
We really should just adopt a node 20 requirement for development. I've used it for dev some and I think it just works.